content/docs/connectors/oauth.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Authentication Through an Generic OAuth 2.0 Provider"
+linkTitle: "Generic OAuth 2.0"
+description: ""
+date: 2021-03-15
+draft: false
+toc: true
+weight: 2050
```

Comment:
This is the same weight as oidc -- which means it will be next to it in the menus. Is this a good position for this page?

Reply:
![Screen Shot 2021-03-15 at 11 43 55 AM](https://user-images.githubusercontent.com/4453979/111205139-d0e6d000-85be-11eb-8610-0cdd1e302eec.png)

Reply:
thx for pointing that out! Didn't think about this value as i took it from OIDC doc directly. 

How is the weight being decided (relevance/importance/users number/time being added)? Though, this connector is indeed close to OIDC connector since OIDC is just a layer on top of oauth 2.0 protocol.

Reply:
Maybe we slot it in above OIDC then, Thoughts @sagikazarmark?

```suggestion
weight: 2045
```

Reply:
Sorry for interfering. I thought connectors should be in the order they appear in the table on the index page. If you want it to be above OIDC, you need to change it's the order in the table as well.

Reply:
Personally, I'd like to see this list sorted by importance (arbitrary) and alphabetical. Eg.:

- OIDC
- OAuth 2.0
- SAML
- LDAP
- Provider specific connectors in alphabetical order

So I guess for now it's fine under OIDC.

Reply:
Set it to `2055` so it goes under OIDC. Also positioned it under OIDC in the connectors table.